lib/wallet/wallet.js

```diff
@@ -4925,6 +4925,35 @@ class Wallet extends EventEmitter {
   static isWallet(obj) {
     return obj instanceof Wallet;
   }
+
+  /**
+   * Clear a cache.
+   * @param {String} cacheName
+   * @param {String | undefined} cacheKey
+   */
+
+   clearCache(cacheName, cacheKey) {
+      var cache;
+      switch(cacheName) {
+      case "bid":
+        cache = this.sendBidResults;
+        break;
+      case "open":
+        cache = this.sendOpenResults;
+        break;
+      case "update":
+        cache = this.sendUpdateResults;
+        break;
+      default:
+        throw new Error(`invalid cache name: ${cacheName}`)
+      }
+
+      if (!cacheKey) {
```

Comment:
doesn't matter at all given our keys, but as a general comment about this type of check since it may apply in other code in the future, would use an explicit undefined or null check here since the empty string is falsey but could be a valid key

Reply:
I was including the empty string in the set of desirably false-y keys, but can check for it explicitly.